doc/rofilib/configurations/parsing_in_cpp.rst

```diff
@@ -0,0 +1,85 @@
+Parsing Configurations in Cpp
```

Comment:
Is there some reason why `C++` is not used?

Reply:
I think it could be used - I wanted to not use special symbols in headers